client/my-sites/upgrades/cart/cart-item.jsx

```diff
@@ -51,6 +52,27 @@ module.exports = React.createClass( {
 		} );
 	},
 
+	monthlyPrice: function() {
+		const { cost, product_slug: productSlug, currency } = this.props.cartItem,
+			// A hacky way to check if we are in signup, since for some reason cartItem.extra.context
+			// is missing in that case (it seems cartItem.extra is overwritten somewhere between)
+			isInSignup = this.props.cartItem.extra && this.props.cartItem.extra.context !== 'calypstore',
+			isPaidPlan = productSlug === 'value_bundle' || productSlug === 'business-bundle';
+		if ( ! isInSignup ||
+				! isPaidPlan ||
+				abtest( 'monthlyPlanPricing' ) === 'yearly' ||
+				cost === 0 ) {
+			return null;
+		}
+
+		return this.translate( '(%(monthlyPrice)s %(currency)s x 12 months)', {
```

Comment:
Good suggestion - I've used `%(monthlyPrice)f` though, so that the decimal part won't get cut off :)